book/src/kubernetes-changelog.md

```diff
@@ -27,19 +37,19 @@ details on individual features, visit the [Features section](features.md).
 * `csi.storage.k8s.io/v1alpha1` `CSINodeInfo` and `CSIDriver` CRDs are no longer supported.
 
 ### Features
-* New beta features:
+* Beta
     * Topology
     * Raw block
     * Skip attach
     * Pod info on mount
-* New alpha features:
+* Alpha
     * Volume expansion
-* New `storage.k8s.io/v1beta1` `CSINode` and `CSIDriver` objects were introduced.
+* `storage.k8s.io/v1beta1` `CSINode` and `CSIDriver` objects introduced.
 
 ## Kubernetes 1.13
 
 ### Deprecations
-* CSI spec 0.2 and 0.3 are deprecated and support will be removed in Kubernetes 1.18.
```

Comment:
hmmm :-)

Reply:
yeah that was my mistake... the k8s release notes actually said removal in 1.16...